rootfs/scheduler/__init__.py

```diff
@@ -612,7 +687,19 @@ def pod_state(self, pod):
 
     def _api(self, tmpl, *args):
         """Return a fully-qualified Kubernetes API URL from a string template with args."""
-        url = "/api/{}".format(self.apiversion) + tmpl.format(*args)
+        # FIXME better way of determining API version based on requested component
+        # extensions use apis and not api
+        # TODO this needs to be aware that deployments / rs could be top level in future releases
```

Comment:
Not yet, I'll make create a ticket - I have a fix in mind, but it isn't urgent enough